modules/oc-mirror-migration-differences.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * oc-mirror-migration-v1-to-v2.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="oc-mirror-migration-differences_{context}"]
+= Changes from oc-mirror plugin v1 to v2
+
+Before migrating from oc-mirror plugin v1 to v2, you can check differences between oc-mirror plugin v1 to v2 below:
+
+. *Explicit version selection*: Users must explicitly specify `--v2` when using `oc-mirror`. If no version is specified, v1 is executed by default. This behavior is expected to change in future releases (4.19 or later), where `--v2` will be the default.
+
+. Commands have changed to align with oc-mirror plugin v2's new workflow.
+
+.. For mirror-to-disk, run the following command:
++
+[source,terminal]
+----
+$ oc-mirror --config isc.yaml file://some-dir --v2
+----
+
+.. For disk-to-mirror, run the following command:
++
+[source,terminal]
+----
+$ oc-mirror --config isc.yaml --from file://some-dir docker://remote-registry --v2
+----
+
+.. For mirror-to-mirror, run the following command:
++
+[source,terminal]
+----
+$ oc-mirror --config isc.yaml --workspace file://some-dir docker://remote-registry --v2
+----
++
+[NOTE]
+====
+`--workspace` is now required for mirror-to-mirror operation.
+====
+
+. *API version update*: The `ImageSetConfiguration` API version changes from `v1alpha2` (v1) to `v2alpha1` (v2). You must manually update their configuration files before migration.
+. *Configuration changes*: 
+- `storageConfig` must be removed in oc-mirror plugin v2.
+- Incremental mirroring is now handled automatically through the working directory/local cache.
+ *Changes in results directory*: all custom resources to be applied to the disconnected cluster are generated in the `<workspace_path>/working-dir/cluster-resources` directory after the migration.
+- Outputs in oc-mirror plugin v2 are not stored in the same location as oc-mirror plugin v1.
+- You must check the cluster-resources folder under the working directory for:
+** IDMS (`ImageDigestMirrorSet`)
+** ITMS (`ImageTagMirrorSet`)
+** `CatalogSource`
+** `ClusterCatalog`
+** `UpdateService`
+
+. *Workspace and directory naming*: Follow the new oc-mirror v2 convention to prevent any potential data inconsistencies when transitioning between versions.
+- The v1 `oc-mirror-workspace` is no longer needed.
+- Use a new directory for oc-mirror plugin v2 to avoid conflicts.
```

Comment:
To be safe better to highlight the above two points about `workspace no longer needed` and use new dir for oc-mirror plugin v2 to avoid conflicts.